src/service.c

```diff
@@ -494,6 +494,16 @@ create_header_state (IndicatorPowerService * self)
           g_object_unref (icon);
         }
     }
+    else if (ayatana_common_utils_is_lomiri())
```

Comment:
So, this always kicks in if Lomiri runs on a device without reported batteries. As we are mainlining Lomir + Indicators to be suitable for desktop and touch based systems for any distro, so also with the use case of running on desktops without batteries attached, we should probably find a way to check if we are in the PDK environment or similar. Right?

Reply:
> So, this always kicks in if Lomiri runs on a device without reported batteries. As we are mainlining Lomir + Indicators to be suitable for desktop and touch based systems for any distro, so also with the use case of running on desktops without batteries attached, we should probably find a way to check if we are in the PDK environment or similar. Right?

That should not be necessary. The newly added case does not control the visibility of the indicator. We are only exporting all the header properties to D-Bus. The visibility is controled by GSettings here:

https://github.com/AyatanaIndicators/ayatana-indicator-power/blob/8a223541810566c2c5ccc5d4f7f9f0b2425bf0ca/src/service.c#L452-L453

and here:

https://github.com/AyatanaIndicators/ayatana-indicator-power/blob/8a223541810566c2c5ccc5d4f7f9f0b2425bf0ca/src/service.c#L402-L438

Under normal circumstances **no batteries == no indicator**.

Reply:
I currently have a VM here, that has no battery. And Lomiri shows the indicator nonetheless (as cogwheel).
![image](https://user-images.githubusercontent.com/407231/221998705-ed7a2cc2-993b-4dda-bc3b-f2938cefb1b2.png)